brave/brave-core
# feat(wallet): Activity Tab V2

## Description 
Updates the `Activity` tab to the new `V2` designs


Resolves <https://github.com/brave/brave-browser/issues/31316>
Resolves <https://github.com/brave/brave-browser/issues/32546>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Create all types of transactions 
- Send (Filecoin, Ethereum, Solana, ERC20, ERC721, SPL)
- Swap (Ethereum, Solana)
- ERC20 Approval
- ETC..
2. Go to the `Activity` tab on the `Page` and the `Panel
3. Check and make sure all `Information` is accurate and is displaying correctly.
4. Click to view `Details` for all types of transaction, make sure all `Information` is accurate and is displaying correctly.
5. Test the `Search` bar and make sure it is filtering correctly.
6. Note: (At the moment we are not able to display token information for `Solana` swaps.)

![Screenshot 33](https://github.com/brave/brave-core/assets/40611140/2726a439-a1e3-4bcd-89b9-a4fdd4c3ce15)

![Screenshot 34](https://github.com/brave/brave-core/assets/40611140/4c3e6fad-c450-4f3c-ae6b-ad3c4c386e67)

![Screenshot 35](https://github.com/brave/brave-core/assets/40611140/04e7d947-4dd7-4094-8e54-15fa3692755e)

![Screenshot 36](https://github.com/brave/brave-core/assets/40611140/f3b51e91-ee46-454d-a4c4-9feffb2ed2b1)

![Screenshot 37](https://github.com/brave/brave-core/assets/40611140/c6078f6e-0181-40c3-96f1-672e2f173bfa)

![Screenshot 38](https://github.com/brave/brave-core/assets/40611140/c7fb8f83-f7b9-4951-9feb-ba7978ab1466)

![Screenshot 39](https://github.com/brave/brave-core/assets/40611140/751eb78d-fa77-495e-8d3c-ea1b5a6232b0)

![Screenshot 40](https://github.com/brave/brave-core/assets/40611140/6854c8dd-84f8-4d16-9caa-04c5c197977b)

![Screenshot 41](https://github.com/brave/brave-core/assets/40611140/6a3047e5-9ca8-44b0-96eb-f2a31bdd904b)

![Screenshot 42](https://github.com/brave/brave-core/assets/40611140/90bb1ebe-31ec-43cb-8d06-c32b56c7039a)

![Screenshot 43](https://github.com/brave/brave-core/assets/40611140/b4028700-1fc4-4e49-877e-a963b5e3856d)

![Screenshot 44](https://github.com/brave/brave-core/assets/40611140/a5503d53-02bb-4863-93b6-3586d971ba95)
